core/reporter-impl/src/test/java/org/arquillian/reporter/impl/asserts/ReportAssert.java

```diff
@@ -151,6 +149,35 @@ public REPORTASSERTTYPE hasEntriesContaining(List<Entry> expectedEntries) {
         return (REPORTASSERTTYPE) this;
     }
 
+    public REPORTASSERTTYPE hasEntryContainingKeys(StringKey... expectedStringKey) {
+        return hasEntryContainingKeys(Arrays.asList(expectedStringKey));
+    }
+
+    public REPORTASSERTTYPE hasEntryContainingKeys(List<StringKey> expectedStringKeys) {
```

Comment:
These two methods are for KeyValue entries or for all entries? According to the implementation, it seems to be only for key-values - please change the method name appropriately.